js-api/src/chem.ts

```diff
@@ -158,11 +150,17 @@ export namespace chem {
     resized = false;
     _sketcherTypeChanged = false;
     _autoResized = true;
+    _type = currentSketcherType;
```

Comment:
How many fields and properties do we need? There is _type, sketcherType, currentSketcherType. Why so many?

Reply:
Since saving this _type is onle neede for substructure filters synchronization, I've removed both _type and sketcherType getter from Sketcher object. Save it on substructure filter side.